airflow/providers/amazon/aws/operators/redshift_to_s3.py

```diff
@@ -107,7 +112,12 @@ def execute(self, context):
         credentials = s3_hook.get_credentials()
         unload_options = '\n\t\t\t'.join(self.unload_options)
         s3_key = '{}/{}_'.format(self.s3_key, self.table) if self.table_as_file_name else self.s3_key
-        select_query = "SELECT * FROM {schema}.{table}".format(schema=self.schema, table=self.table)
+        if self.custom_select_query is None:
```

Comment:
We need  a test case covering that case. there is a test already in TestRedshiftToS3Transfer and you can add new one with custom select query

Reply:
Yes, I will work on adding the case, Thank you!